config_params.py

```diff
@@ -83,7 +83,7 @@ class RasterStatus(Enum):
 DEWAR_SECTORS = {"amx": 8, "fmx": 8, "nyx": 5}
 PUCKS_PER_DEWAR_SECTOR = {"amx": 3, "fmx": 3, "nyx": 3}
 
-cryostreamTempPV = {"amx": "AMX:cs700:gasT-I", "fmx": "FMX:cs700:gasT-I"}
+cryostreamTempPV = {"amx": "XF:17IDB-ES:AMX{CS:1}SAMPLE_TEMP_RBV", "fmx": "FMX:cs700:gasT-I"}
```

Comment:
FMX's PB does not appear to be working right now (March 8, 10:23am) - can this be updated now?

Reply:
any update on the actual PV that works for FMX?

Reply:
No, FMX does not have a working system, they are going to install it during the upcoming shutdown in which case I will update this PV

Reply:
good enough for now then...